File: articles/algorithmic-fairness-and-computational-justice/java/FairnessGapScore.java
public class FairnessGapScore {
    public static void main(String[] args) {
        double[] rates = {0.42, 0.31, 0.36};
        double minRate = rates[0];
        double maxRate = rates[0];
        for (double value : rates) {
            if (value < minRate) minRate = value;
            if (value > maxRate) maxRate = value;
        }
        System.out.printf("selection_gap=%.4f%n", maxRate - minRate);
    }
}

File: articles/algorithmic-fairness-and-computational-justice/rust/fairness_gap_score.rs
fn main() {
    let rates = [0.42_f64, 0.31_f64, 0.36_f64];
    let max_rate = rates.iter().cloned().fold(f64::NEG_INFINITY, f64::max);
    let min_rate = rates.iter().cloned().fold(f64::INFINITY, f64::min);
    let gap = max_rate - min_rate;
    println!("selection_gap={:.4}", gap);
}
